src/tree-view/interfaces.ts

```diff
@@ -90,6 +95,8 @@ export namespace TreeViewProps {
     expanded: boolean;
   }
 
+  export type ConnectorLinesVariant = 'vertical';
```

Comment:
Should we use 'vertical' or `'vertical' | 'none'`? If the latter - we should also assign the default in index.tsx. Also, let's add this prop to the metrics.